src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -32,6 +32,11 @@
     protected static final String MAIN_AUTHOR_NAME = "harryggg";
     protected static final String FAKE_AUTHOR_NAME = "fakeAuthor";
     protected static final String EUGENE_AUTHOR_NAME = "eugenepeh";
+    protected static final String LATEST_EUGENE_AUTHOR_COMMIT_HASH = "2d87a431fcbb8f73a731b6df0fcbee962c85c250";
```

Comment:
latest is subjective though, it might change as we update the test repo.
How about `COMMIT_HASH_EUGENE_DD_MM_YYYY`?